src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -136,10 +136,13 @@ const Allergies = props => {
     }
     if (allergies?.length === 0) {
       return (
-        <div className="vads-u-margin-bottom--3">
-          <va-alert background-only status="info">
-            You don’t have any records in Allergies
-          </va-alert>
+        <div
+          className="record-list-item vads-u-border-color--gray-light vads-u-border--0 vads-u-background-color--gray-lightest card"
+          data-testid="record-list-item"
+        >
+          <h2 className="vads-u-font-size--base vads-u-font-weight--normal vads-u-font-family--sans no-margin-top ">
```

Comment:
Rather than define a custom `no-margin-top` style, can you use the design library's `vads-u-margin-top--0` style?